meta-lmp-bsp/recipes-support/mfgtool-files/mfgtool-files/apalis-imx8-sec/close.uuu

```diff
@@ -0,0 +1,44 @@
+uuu_version 1.3.102
+
+SDPS: boot -f imx-boot-mfgtool
```

Comment:
should this be .signed? so it can verify the AHAB status before closing (I don't remember if that is possible with AHAB)
```suggestion
SDPS: boot -f imx-boot-mfgtool.signed
```